#!/usr/bin/env python
PACKAGE = "pointcloud_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("leaf_size", double_t, 0, "leaf_size",    .05, 0.01,   0.2)

exit(gen.generate(PACKAGE, "pointcloud_filter", "VoxelGridFilter"))
